### PR TITLE
[95692838] Export .aws_credentials in standard and terraform forms

### PR DESCRIPTION
**Pull request by mtekel**
Adjust .aws_credentials file (defined in the vault) so that it
1. Exports AWS credentials in a standard form (`AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`). Adjust ansible-deploy job to consume them directly.
2. Exports AWS credentials in a terraform form (`TF_VAR_aws_access_key` and `TF_VAR_aws_secret_key`). Adjust terraform job to consume them directly.

### CODE REVIEW

File: jobs/terraform-deploy/terraform-deploy.groovy [hunk 1]
```diff
@@ -46,8 +46,7 @@ terraform ${action} -var env=${environment_name} -input=false ${extraopts}
 # AWS
 cd ../aws
 . ~/.aws_credentials
-terraform ${action} -var env=${environment_name} -input=false ${extraopts} \
--var "aws_access_key=${aws_access_key}" -var "aws_secret_key=${aws_secret_key}"
+terraform ${action} -var env=${environment_name} -input=false ${extraopts}
```
Comment (dcarley) [inline jobs/terraform-deploy/terraform-deploy.groovy:49]: It feels to me like the following might be easier to understand, wouldn't result in duplicate variables in the vault, and would protect us if the variables were renamed in alphagov/tsuru-terraform (I wouldn't think to check here):

```
-var "aws_access_key=${AWS_SECRET_ACCESS_KEY}" -var "aws_secret_key=${AWS_SECRET_ACCESS_KEY}"
```

What do you think?

Comment (mtekel) [inline jobs/terraform-deploy/terraform-deploy.groovy:49]: If you rename variables in tsuru-terraform then you would have to update this line anyway. Not providing variables for AWS via -var unifies the command with GCE terraform.

Comment (dcarley) [inline jobs/terraform-deploy/terraform-deploy.groovy:49]: Good point. Although they are visible in the job, as opposed to hidden in the vault.

I'm not sure whether unifying them is that important. We could wrap them with a `Makefile` like we have for Ansible.

**Comment by actionjack** [comment]
LGTM :+1:

**Pull request merged**